packages/api/db/functions.sql

```diff
@@ -63,6 +63,7 @@ BEGIN
             (data ->> 'files')::jsonb,
             (data ->> 'origins')::jsonb,
             (data ->> 'meta')::jsonb,
+            json_arr_to_text_arr(data -> 'backup_urls'),
             (data ->> 'updated_at')::timestamptz,
             (data ->> 'inserted_at')::timestamptz)
     ON CONFLICT ( user_id, source_cid )
```

Comment:
Need to append the new backup URLs `ON CONFLICT`.

Reply:
sorry didnt get this one ? it needs backup_urls in the on conflict clause ?

Reply:
Will do

Reply:
Yeah also had a Q about this - the on conflict is handling a case where a user is uploading an identical file for a second time, wouldn't the backup url also stay the same?